flask_restful/__init__.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, app=None, prefix='',
         self.catch_all_404s = catch_all_404s
         self.url_part_order = url_part_order
         self.errors = errors or {}
+        self.errorhandlers = []
```

Comment:
Oh, I see. I was looking at the `App.error_handler_spec` in Flask, which is a `dict`, but thats storing lists of errorhandlers keyed by blueprint name.